repos/spack_repo/builtin/packages/mpb/package.py

```diff
@@ -30,6 +31,11 @@ class Mpb(AutotoolsPackage):
     depends_on("hdf5")
     depends_on("libctl")
 
+    def flag_handler(self, name, flags):
+        if self.spec.satisfies("%c=gcc@14:") and name == "cflags":
+            flags.append("-Wno-int-conversion")
```

Comment:
It might be less computationally intensive to switch the ordering of these two conditions.

Reply:
Amdahl's law probably says don't bother when the build takes minutes, but I fixed it anyway.